app/layout/projects/show/map/index.tsx

```diff
@@ -131,18 +134,44 @@ export const ProjectMap: React.FC<ProjectMapProps> = () => {
     },
   });
 
-  const LAYERS = [PUGridLayer, PUCompareLayer, PlanningAreaLayer].filter((l) => !!l);
+  const FeaturePreviewLayers = useFeaturePreviewLayers({
+    features: selectedFeaturesData,
+    active: tab === 'features',
```

Comment:
Following [this comment](https://github.com/Vizzuality/marxan-cloud/pull/1417/files#r1284048624), same approach here: while there are selected features, the layer should be considered _active_.